app/src/main/java/com/codesoom/assignment/task/service/TaskServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.codesoom.assignment.task.service;
+
+import com.codesoom.assignment.task.domain.Task;
+import com.codesoom.assignment.task.domain.request.TaskSearchDto;
+import com.codesoom.assignment.task.repository.TaskRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class TaskServiceImpl implements TaskService{
```

Comment:
```suggestion
public class TaskServiceImpl implements TaskService {
```
nit : 아주 사소하지만 일관성있게 정리된 것이 낫죠!

Reply:
아 맞습니다!
어제 급하게 PR 올리려고 부랴부랴 코딩하느라 놓쳤나보네요..!
감사합니다 :)